k8s/base.py

```diff
@@ -186,6 +186,13 @@ def delete(cls, name, namespace="default", **kwargs):
         url = cls._build_url(name=name, namespace=namespace)
         cls._client.delete(url, **kwargs)
 
+    @classmethod
+    def delete_list(cls, namespace="default", labels=None, **kwargs):
```

Comment:
Any reason to take `kwargs` here? It's not used.

Reply:
Good spot - I've changed it so now it's passed to the client call, like the other methods here